server/utils/cloudFunctions/functions/test/countAdvertisersTests/countAdvertisersFunctions.test.js

```diff
@@ -0,0 +1,400 @@
+        return wrapper(snapTwo).then(() => {
+          DEV_AGGREGATES_COLLECTION
+              .doc("2020")
+              .collection("advertisers")
+              .doc(advertiser).get().then(function(doc) {
+                return assert.equal(doc.data().numberOfAds, 1);
+              }).catch(err => console.log(err));
+          
+          DEV_AGGREGATES_COLLECTION
+              .doc("2020")
+              .collection("advertisers")
+              .doc(advertiser).get().then(function(doc) {
+                return assert.equal(doc.data().numberOfAds, 1);
+              }).catch(err => console.log(err));
+
+          return;
+        });
+      }); 
+    });   
+
+    it(SHOULD_INCREMENT + 'only for an advertisers own ads', () => {
+      const snapOne = 
+          snapFromJson({advertiser: "adv_f", startDate: "2019-10-15"}, 
+                        DEV_ADS_PATH);
+      const snapTwo = 
+          snapFromJson({advertiser: "adv_g", startDate: "2020-10-15"}, 
+                        DEV_ADS_PATH);
+
+      const wrapper = firestoreWrap(devCountAdvertisersOnCreate);
+      
+      return wrapper(snapOne).then(() => {
+        return wrapper(snapTwo).then(() => {
+          DEV_AGGREGATES_COLLECTION
+              .doc("2019")
+              .collection("advertisers")
+              .doc("adv_f").get().then(function(doc) {
+                return assert.equal(doc.data().numberOfAds, 1);
+              }).catch(err => console.log(err));
+          
+          DEV_AGGREGATES_COLLECTION
+              .doc("2020")
+              .collection("advertisers")
+              .doc("adv_g").get().then(function(doc) {
+                return assert.equal(doc.data().numberOfAds, 1);
+              }).catch(err => console.log(err));
+          
+          return;
+        });
+      }); 
+    });
+
+    it("should fail with malformed inputs", () => {
+      const snapOne = 
+          snapFromJson({advertiser: 101, startDate: "2019-10-15"}, 
+                        DEV_ADS_PATH);
+      const snapTwo = 
+          snapFromJson({advertiser: "adv", startDate: 2019}, DEV_ADS_PATH);
+      const snapThree = 
+          snapFromJson({advertiser: "adv", startDate: "201-910-15"}, 
+                        DEV_ADS_PATH);
+
+      const wrapper = firestoreWrap(devCountAdvertisersOnCreate);
+
+      assert.throws(() => wrapper(snapOne), Error);
+      assert.throws(() => wrapper(snapTwo), Error);
+      assert.throws(() => wrapper(snapThree), Error);
+      assert.throws(() => wrapper(snapFour), Error);
+    });  
+  });
+  
+  describe("test_countAdvertisersOnDelete", () => {
+    it("should decrement ad count when deleting an advertisers ad", () => {
+      // The 'after' should cleanup the DB of "adv_a" references and values
+      // in time, but it can be flaky. 
+      // Easy way to avoid this problem is just to switch to adv_A, adv_B, 
+      // and so on. 
+      const snap = snapFromJson({advertiser: "adv_A", startDate: "2018-10-15"}, 
```

Comment:
be consistent in how you write these tests.  In the others you declare advertiser separately rather than inline.